is_core/main.py

```diff
@@ -444,6 +445,9 @@ def get_ui_form_field_labels(self, request):
     def get_ui_list_field_labels(self, request):
         return self.ui_list_field_labels if self.ui_list_field_labels is not None else self.get_field_labels(request)
 
+    def get_detail_export_types(self):
+        return self.detail_export_types
+
```

Comment:
nechybí tu get_export_display?

Reply:
to ma core, ve view pouzivam jenom _get_export_display, nevidim duvod aby to ve view bylo public